### PR TITLE
Added extra args to logging fluentd and pass it to config check

### PR DESCRIPTION
**Pull request by petergvizd**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?

Added extra args to logging fluentd and pass it config check
Pass env variables from logging fluentd also to config check

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

**Review by ahma — DISMISSED** [review]
lgtm Thanks @petergvizd

File: pkg/resources/fluentd/statefulset.go [hunk 1]
```diff
@@ -149,6 +149,10 @@ func fluentContainer(spec *v1beta1.FluentdSpec) corev1.Container {
 		}
 	}
 
+	if len(spec.ExtraArgs) != 0 {
+		container.Args = append(container.Args, spec.ExtraArgs...)
+	}
```
Comment (siliconbrain) [inline pkg/resources/fluentd/statefulset.go:154]:
```suggestion
	container.Args = append(container.Args, spec.ExtraArgs...)
```

File: pkg/resources/fluentd/appconfigmap.go [hunk 2]
```diff
@@ -208,6 +208,16 @@ func (r *Reconciler) newCheckOutputSecret(hashKey string) (*corev1.Secret, error
...
 func (r *Reconciler) newCheckPod(hashKey string) *corev1.Pod {
+	containerArgs := []string{
+		"fluentd", "-c",
+		fmt.Sprintf("/fluentd/etc/%s", ConfigKey),
+		"--dry-run",
+	}
+
+	if len(r.Logging.Spec.FluentdSpec.ExtraArgs) != 0 {
+		containerArgs = append(containerArgs, r.Logging.Spec.FluentdSpec.ExtraArgs...)
+	}
```
Comment (siliconbrain) [inline pkg/resources/fluentd/appconfigmap.go:219]:
```suggestion
	containerArgs = append(containerArgs, r.Logging.Spec.FluentdSpec.ExtraArgs...)
```

**Review by siliconbrain — APPROVED** [review]
lgtm

**Review by ahma — APPROVED** [review]
lgtm

**Pull request merged**